### PR TITLE
doc: update chart/README.md

### PR DESCRIPTION
**Pull request by mantissahz**
Updated new fields introduced.

Issues: longhorn/longhorn#2997
PR: #6157

### CODE REVIEW

File: chart/README.md [hunk 1]
```diff
@@ -268,6 +269,7 @@ For more details like types or options, you can refer to **Settings Reference**
 | defaultSettings.allowEmptyNodeSelectorVolume | Allow Scheduling Empty Node Selector Volumes To Any Node |
 | defaultSettings.allowRecurringJobWhileVolumeDetached | If this setting is enabled, Longhorn will automatically attaches the volume and takes snapshot/backup  when it is the time to do recurring snapshot/backup. |
 | defaultSettings.allowVolumeCreationWithDegradedAvailability | This setting allows user to create and attach a volume that doesn't have all the replicas scheduled at the time of creation. |
+| defaultSettings.autoCleanupRecurringJobBackupSnapshot | This setting enables Longhorn to automatically cleanup the snapshot generated by a recurring backup job. |
```
Comment (innobead) [inline chart/README.md:272]: Is it incorrect.

**Review by innobead — CHANGES_REQUESTED** [review]

File: chart/README.md (hunk 1 above)
Comment (mantissahz) [reply chart/README.md:272]: That is the description of the `defaultSettings.autoCleanupRecurringJobBackupSnapshot` in question.yaml

Comment (innobead) [reply chart/README.md:272]: Why we have seevice monitor value in this PR as well?

Comment (mantissahz) [reply chart/README.md:272]: This PR is trying to update the README.md for all missing parts.
And we missed these two parts by accident.
Do you consider that we need to separate them into two PRs?

Comment (innobead) [reply chart/README.md:272]: oh I see. It's better to describe it clearly in the PR description.

**Review by innobead — APPROVED** [review]

**Pull request merged**